### DISCUSSION TITLE
Addition of an `--ignore-errors` flag

### DISCUSSION THREAD

**Discussion by mortonmills** [jgm/pandoc#9858] [Ideas]
I've been using pandoc to convert repo docs (markdown, pandoc, node, vscode.... ) into html and epubs for offline use.
However, I receive errors in some cases and only receive partial output.

I've searched the pandoc issues and discussions, 
#4893 requested this as an option several years ago.

I realize the current solution is to use filters.
However, having an `--ignore-errors` flag would greatly simplify usage when errors are acceptable 
and could also act as a last resort option in the case filters won't do or are too complex to write.

**Comment by tarleb** [comment]
In general, errors are throws if there is no reasonable way to continue with the execution. Specific cases where a warning might suffice should be looked at individually.

**Comment by Disonantemus** [comment]
I also would like to have `--ignore-errors` flag, my use case:
Today I did need to convert a markdown file to PDF, it wasn't created because this error:

~~~ocaml
$ pandoc gfm.md -s -t typst -o gfm.pdf
error: file not found (searched at /home/user/Markup/1/home/user/Images/Misc/240px-Instagram_icon.png)
    ┌─ toPdfViaTempFile100364-0.html:356:11
    │
356 │ #box(image("/home/user/Images/Misc/240px-Instagram_icon.png"))
    │            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Error producing PDF.
~~~

I didn't care about the error (missing image file), but if I have the flag, don't need to edit the markdown file to get PDF created.

**Reply by jgm** [reply]
That error is raised by typst itself, not pandoc.
